src/components/ivq-overlay/ivq-overlay.scss

```diff
@@ -5,6 +5,10 @@
 }
 
 .ivqOverlay {
+  z-index: 1;
+  position: absolute;
```

Comment:
if you use absolute and width: 100%, height: 100% add top: 0, left: 0.
I remember that some browsers has an issue if you not define coordinates